providers/os/connection/snapshot/blockdevices.go

```diff
@@ -53,9 +76,43 @@ func (cmdRunner *LocalCommandRunner) GetBlockDevices() (*BlockDevices, error) {
 	if err := json.Unmarshal(data, blockEntries); err != nil {
 		return nil, err
 	}
+	blockEntries.FindAliases()
+
 	return blockEntries, nil
 }
 
+func (blockEntries *BlockDevices) FindAliases() {
+	entries, err := os.ReadDir("/dev")
+	if err != nil {
+		log.Warn().Err(err).Msg("Can't read /dev directory")
+		return
+	}
+
+process_symlinks:
```

Comment:
if we were to read the entries and then pass those to a function that looks like `findAlias(block BlockDevice, entries []fs.Info)`, would that eliminate the need for the labeled loop? i think we should avoid this